applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -188,65 +166,88 @@ int le_audio_user_defined_button_press(enum le_audio_user_defined_action action)
 int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay);
 
 /**
- * @brief	Increase volume by one step
+ * @brief	Set pointer to connection, used by CIS.
```

Comment:
```suggestion
 * @brief	Set pointer to the connection, used by CIS.
```
Connection used by CIS or the function is used by CIS? This is a global issue. Expand the acronym.

Reply:
I've added a note for ´Used by CIS´. I do not agree that CIS should be expanded, what CIS and BIS are is explained in the README